i18n/po/8311.fr.po

```diff
@@ -423,6 +435,12 @@ msgstr "Adresse MAC LCT"
 msgid "MAC address of the LCT management interface (XX:XX:XX:XX:XX:XX format)."
 msgstr "Adresse MAC de l'interface de gestion LCT (format XX:XX:XX:XX:XX:XX)."
 
+msgid "Reverse ARP Monitoring"
+msgstr "Inverser le monitoring ARP"
```

Comment:
Is it "Reverse the ARP Monitoring" or "enable Monitoring for Reverse ARP protocol" ?